src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CodeFoldingTests.cs

```diff
@@ -57,7 +57,7 @@ private async Task AssertFoldableBlocksAsync(params string[] blockTexts)
                     return;
                 }
 
-                Thread.Sleep(500);
+                await Task.Delay(500);
```

Comment:
This ends up being better in the test to make sure we don't block any specific thread. Thread.Sleep was causing issues on occasion with local testing.